eqcorrscan/core/lag_calc.py

```diff
@@ -227,17 +236,18 @@ def _channel_loop(detection, template, min_cc, detection_id, interpolate, i,
             waveform_id=_waveform_id, time=picktime,
             method_id=ResourceIdentifier('EQcorrscan'), phase_hint=phase,
             creation_info='eqcorrscan.core.lag_calc',
+            evaluation_mode='automatic',
             comments=[Comment(text='cc_max=%s' % cc_max)]))
         event.resource_id = detection_id
     ccc_str = ("detect_val=%s" % cccsum)
     event.comments.append(Comment(text=ccc_str))
     if used_chans == detect_chans:
         if pre_lag_ccsum is not None and\
-           checksum - pre_lag_ccsum < -(0.30 * pre_lag_ccsum):
+           checksum - pre_lag_ccsum < -(0.3 * pre_lag_ccsum):
             msg = ('lag-calc has decreased cccsum from %f to %f - '
                    % (pre_lag_ccsum, checksum))
-            # warnings.warn(msg)
-            raise LagCalcError(msg)
+            warnings.warn(msg)
```

Comment:
Changing this results in failed tests - lag-calc should never reduce the correlation sum if the same channels are used. What was the reason for changing this?